include/drivers/clock_control/nrf_clock_control.h

```diff
@@ -38,7 +38,7 @@ enum clock_control_nrf_type {
 /** @brief LF clock start modes. */
 enum nrf_lfclk_start_mode {
 	NRF_LFCLK_START_MODE_NOWAIT,
-	NRF_LFCLK_START_MODE_SPINWAIT_RUNNING,
+	NRF_LFCLK_START_MODE_SPINWAIT_AVAILABLE,
 	NRF_LFCLK_START_MODE_SPINWAIT_STABLE,
 };
```

Comment:
Shouldn't we use other prefixes here? I know, `CLOCK_CONTROL_NRF_` is lengthy, but the current one is not consistent with the other stuff defined in this file.

Reply:
renamed to `CLOCK_CONTROL_NRF_LF_START_[STABLE, AVAILABLE,NOWAIT]`